src/Boxer/main.cpp

```diff
@@ -7,6 +7,8 @@
 #ifdef PLATFORM_WINDOWS
 #include <crtdbg.h> // include to bring _CrtSetDbgFlag into scope
 #endif
+#include "Math/Math.h"
+

```

Comment:
...Do I want to know why main.cpp is including math? It definitely should not be.